src/jit/codegencommon.cpp

```diff
@@ -9530,6 +9542,15 @@ void                CodeGen::genFnEpilog(BasicBlock* block)
                 BAD_IL_OFFSET, REG_NA, REG_NA, 0, 0, /* iloffset, ireg, xreg, xmul, disp */
                 true);                     /* isJump */
         }
+#if FEATURE_FASTTAILCALL
+        else
+        {
+            // Fast tail call.
+            // Call target = REG_IP0.
+            // TODO: Do we need a special encoding for stack walker like rex.w prefix for x64?
```

Comment:
Please file a Git issue to track this TODO. I think to support fast tail calls VM side support might be needed. It would be good to track that work with an issue.